css-two-column-layout/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-whole">
+        <h2>SCALE, SPEED, AGILITY</h2>
+        <p>We've dedicated thousands of man hours to making healthcare institutins in the Helium network run 10x more efficiently and deliver an unparallel quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half"></div>
+      <img src="images/software.svg" alt="software">
+      <div class="text">
+        <h4>INTUITIVE,<br>
+          SMART SOFTWARE</h4>
+        <p>We build products that transform how healthcare facilities run across Africa - making them radically more efficient
+          operationally and financially, a complete turnaround.</p>
+      </div>
+      <div class="column-half"></div>
+      <img src="images/practice.svg" alt="medicine">
+      <div class="text">
+        <h4>PRACTICE MEDICINE, NOT
+          <br> ADMINISTRATION</h4>
+        <p>Our all-in-one electronic health records (EHR), hospital management (HMIS) &amp; billing systems helps streamline your
+        facility, reduce waste and improve quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-half"></div>
```

Comment:
You don't have any content inside this `column-half` `div`, you should have all of your content for this column inside here.